### PR TITLE
Release debian golang updates

### PR DESCRIPTION
**Pull request by rcrozean**
*Issue #, if available:*
#1142 
*Description of changes:*
Bump release flag to trigger new debian build with Sept 6th patch release.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by markapruett — APPROVED** [review]

File: projects/golang/go/README.md
Comment (zafs23) [inline projects/golang/go/README.md:1]: can you please also add in line 25 that the artifacts will be available?

**Pull request merged**